aptos-move/aptos-vm/src/move_vm_ext/respawned_session.rs

```diff
@@ -388,8 +388,6 @@ impl<'r> TResourceGroupView for ExecutorViewWithChangeSet<'r> {
             .and_then(|g| g.inner_ops().get(resource_tag))
         {
             randomly_check_layout_matches(maybe_layout, layout.as_deref()).map_err(|e| {
-                // TODO[agg_v2](cleanup): Push into `randomly_check_layout_matches` once Satya's
```

Comment:
We should remove this `map_err` and instead `randomly_check_layout_matches(..)?`

Reply:
Done.